GCMessengerSDKSample/GCMessengerSDKSample/AccountDetailsViewController.swift

```diff
@@ -40,17 +41,19 @@ class AccountDetailsViewController: UIViewController {
         domainIdTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
         
         if let deploymentId = deploymentIdTextField.text, let domainId = domainIdTextField.text {
-            if deploymentId.isEmpty && domainId.isEmpty {
-                startChatButton.isEnabled = false
-            }
+            let domainAndDeploymentIdsAreEmpty = deploymentId.isEmpty && domainId.isEmpty
```

Comment:
please extract to a separate function to avoid duplicity - maybe 'setButtonsAvailability()'